src/utils/tools/zipFolder.ts

```diff
@@ -0,0 +1,14 @@
+import pify from 'pify'
+import zipFolderPackage from 'zip-folder'
+// promise-style version of zip-folder
+const pzipFolder = pify(zipFolderPackage)
+
+async function zipFolder (folderPath: string, outputPath: string) {
+  try {
+    await pzipFolder(folderPath, outputPath)
+  } catch (err) {
+    throw new Error(`Some error occurred while generating the zip file: ${err}`)
+  }
+}
+
```

Comment:
The zipping process was used in `CustomImage` and `customFont` classes, only moved to a separate tool and used a Promise style version.

Reply:
You're right, I will stick with the arrow functions, gonna make the changes.

Reply:
Resolved.